.github/workflows/PR-build.yaml

```diff
@@ -0,0 +1,31 @@
+name: PR build
+
+on:
+    workflow_dispatch:
+    pull_request:
+        branches: [main]
+
+jobs:
+    Build-wrapping-artifacts:
+        name: Build wrapping arifacts
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkin Code
+              uses: actions/checkout@v4
+            
+            - name: Set up JDK 11
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'temurin'
+                cache: maven
+            
+            - name: Building artefacts
+              run: |
+               mvn clean install -Dgpg.skip=true \
+               -Dgalasa.source.repo=https://repo.maven.apache.org/maven2/ \
+               -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+               -Dgalasa.release.repo=https://maven.pkg.github.com/jaydee029/wrapping

```

Comment:
- The mvn command should be `deploy` instead of `clean install` to match the current command in the Tekton pipeline: https://github.com/galasa-dev/automation/blob/main/pipelines/pipelines/wrapping/build-pr.yaml#L125